test/build-test.ts

```diff
@@ -42,7 +42,7 @@ describe("build", () => {
 
       await rm(actualDir, {recursive: true, force: true});
       if (generate) console.warn(`! generating ${expectedDir}`);
-      const config = Object.assign(await readConfig(undefined, path), {output: outputDir});
+      const config = {...await readConfig(undefined, path), output: outputDir};
```

Comment:
This change is to avoid mutating the `Config` instance: the same config is used by other tests.